ImageItem/ImageItem.js

```diff
@@ -88,20 +96,32 @@ const ImageItemBase = kind({
 		className: 'imageItem'
 	},
 
-	render: ({children, css, src, ...rest}) => {
-		const [Component, marqueeProps] = children ? [MarqueeImageItem, {
+	computed: {
+		className: ({captionOverImage, styler}) => styler.append({
+			captionOverImage
+		})
+	},
+
+	render: ({captionOverImage, children, css,  src, ...rest}) => {
+		const imageItemWithOverlayCaption = (children && captionOverImage) ? [UiImageItem, children] : [UiImageItem, null];
+
+		const [Component, marqueeProps] = (children && !captionOverImage) ? [MarqueeImageItem, {
 			alignment: 'center',
 			children
-		}] : [UiImageItem, null];
+		}] : imageItemWithOverlayCaption;
```

Comment:
Can we replace your code with the following code?
```
		const [Component, marqueeProps] = children ? 
			[MarqueeImageItem, {alignment: 'center'}] :
			[UiImageItem, null];
```
In my opinion, it would be more understandable. Please let me know your thought.

Reply:
This code would apply Marquee to caption when placed on image, but we don't want this to happen. This code works:
`const [Component, marqueeProps] = (children && !captionOverImage) ? 
[MarqueeImageItem, {
	alignment: 'center',
	children
}] : [UiImageItem, null];`

Reply:
Ooops. I added the wrong condition. The condition that you wrote is the one that I expected.
By the way, do we have to pass the `children` in `{ alignment: 'center', children }` because there is the duplicated `children` in `<Component ...{ alignment: 'center', children } ... >{children}</Component>`?

Reply:
You're right. We don't need to pass `children` in `{ alignment: 'center', children }`. I will remove it.

Reply:
Good to know~